src/MCPClient/lib/clientScripts/archivematicaCreateProcessedStructmap.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python -OO
+
+from __future__ import print_function
+import os
+import sys
+
+from archivematicaCreateMETS import createFileSec, each_child
+from archivematicaCreateMETS2 import createDigiprovMD
+import archivematicaXMLNamesSpace as ns
+
+from lxml import etree
+
+
+def create_amdSecs(path, file_group_identifier, base_path, base_path_name, sip_uuid):
+    amdSecs = []
+
+    for child in each_child(path, file_group_identifier, base_path, base_path_name, sip_uuid):
+        if isinstance(child, basestring):  # directory
+            amdSecs.extend(create_amdSecs(child, file_group_identifier, base_path, base_path_name, sip_uuid))
+        else:  # file
+            admid = "digiprov-" + child.uuid
+            amdSec = etree.Element(ns.metsBNS + 'amdSec',
+                                   ID=admid)
+            amdSec.extend(createDigiprovMD(child.uuid))
+            amdSecs.append(amdSec)
+
+    return amdSecs
+
+if __name__ == '__main__':
+    from argparse import ArgumentParser
+    parser = ArgumentParser()
+    parser.add_argument("-s", "--basePath", action="store", dest="basePath", default="")
+    parser.add_argument("-b", "--basePathString", action="store", dest="basePathString", default="SIPDirectory")  # transferDirectory
+    parser.add_argument("-f", "--fileGroupIdentifier", action="store", dest="fileGroupIdentifier", default="sipUUID")  # transferUUID
+    parser.add_argument("-S", "--sipUUID", action="store", dest="sipUUID", default="")
+    parser.add_argument("-x", "--xmlFile", action="store", dest="xmlFile", default="")
+    (opts, _) = parser.parse_known_args()
```

Comment:
Is this still necessary, since createMETS and createMETS2 are better able to be imported now?